username/__init__.py

```diff
@@ -1,8 +1,9 @@
 import sys
 import os
+import getpass
 
 class Username():
-	user = os.path.split(os.path.expanduser('~'))[-1]
+	user = os.environ['SUDO_USER'] if 'SUDO_USER' in os.environ else getpass.getuser()
```

Comment:
Yes, I guess that's a better approach, I guess that can be merged in, the only thing left is to drop a new release adding those changes to the [`username-cli`](https://github.com/abranhe/username-cli)